stdlib/3/struct.pyi

```diff
@@ -1,30 +1,35 @@
 # Stubs for struct
 
 # Based on http://docs.python.org/3.2/library/struct.html
+# Based on http://docs.python.org/2/library/struct.html
 
-from typing import overload, Any, Tuple
+import sys
+from typing import Any, Tuple, Text, Union
 
 class error(Exception): ...
 
-def pack(fmt: str, *v: Any) -> bytes: ...
-# TODO buffer type
-def pack_into(fmt: str, buffer: Any, offset: int, *v: Any) -> None: ...
+_FmtType = Union[bytes, Text]
+if sys.version_info >= (3,):
+    _BufferType = Union[bytes, bytearray, memoryview]
```

Comment:
There's a unit test checking that `array.array` is also accepted (http://bugs.python.org/issue1563759). Other objects that implement the C-level buffer protocol should also be affected, but I'm not sure how we can express that in typeshed.